minari/dataset/minari_dataset.py

```diff
@@ -220,7 +176,7 @@ def total_steps(self):
         """Total episodes steps in the Minari dataset."""
         if self._total_steps is None:
             t_steps = self._data.apply(
-                lambda episode: episode["total_steps"],
+                lambda episode: episode.total_steps,
                 episode_indices=self._episode_indices,
             )
             self._total_steps = sum(t_steps)
```

Comment:
Now that `total_steps`is computed at init, you can delete this function.
This was intended for lazy initialization in case of large dataset